source/_docs/configuration/templating.markdown

```diff
@@ -70,7 +70,8 @@ extensions:
 You can write reusable Jinja templates by adding them to a `custom_templates` folder under your
 configuration directory. All template files must have the `.jinja` extension and be less than 5MiB.
 Templates in this folder will be loaded at startup. To reload the templates without
-restarting Home Assistant, invoke the `homeassistant.reload_custom_templates` service.
+restarting Home Assistant, invoke the `homeassistant.reload_custom_templates` service in
+{% my developer_template title="Developer Tools -> Services" %}.
```

Comment:
`developer_template` isn't the right link/place.

Alternatively, maybe just do this:

```suggestion
restarting Home Assistant, invoke the {% my developer_call_service service="homeassistant.reload_custom_templates" %} service.
```

Reply:
Thanks, done.